configuration/configuration.hpp

```diff
@@ -21,5 +21,9 @@ types::SystemTypeMap systemType{
     {"50003000",
      {"50003000_v2", {{"000A", ""}, {"000B", ""}, {"000C", ""}, {"0014", ""}}}},
     {"50004000", {"50004000", {}}}};
+
+std::unordered_map<std::string, std::string> systemSpecificBackupMap{
+    {"50001001", "backup_map_50001001.json"},
+    {"50001001_v2", "backup_map_50001001_v2.json"}};
```

Comment:
Why we need two back up maps for these two. They are just duplicate.

Reply:
removed json for v2, will use same json for v2 system.